src/use_case/send_message/SendMessageOutputData.java

```diff
@@ -0,0 +1,25 @@
+package use_case.send_message;
+
+import java.time.LocalDateTime;
+
+public class SendMessageOutputData {
+
+    String sender;
+
+    LocalDateTime datetime;
+
+    String text;
+
+    public SendMessageOutputData(String text, LocalDateTime datetime, String sender) {
+        this.text = text;
+        this.sender = sender;
+        this.datetime = datetime;
+    }
+
+
+
+
+
+
```

Comment:
No setter and getter?

Reply:
I'll implement the getter. But I believe we won't need any setter since when we construct the OutputData, all the necessary attributes are created and do not need to be altered along their way to the presenter.